backend/api/Controllers/Models/ScheduleLocalizationMissionQuery.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Api.Database.Models;
+
+namespace Api.Controllers.Models
+{
+    public struct ScheduleLocalizationMissionQuery
+    {
+        public string RobotId { get; set; }
+        public Pose LocalizationPose { get; set; }
+        public string DeckId { get; set; }
```

Comment:
At the moment an AssetDeck has one and only one DefaultLocalizationPose, why do we need both a localizationPose and a DeckId if we fetch the AssetDeck from the database? Then we can just read the Pose from the AssetDeck